### PR TITLE
Fix a problem in CopyGroupList + minor style fixes

### PR DESCRIPTION
**Pull request by jianqiao**
This PR adjusts the implementation of `CopyGroupList::merge_contiguous()` to handle some cases correctly.

E.g. let `R`be a split-row-store relation with integer attributes `x`,`y`,`z`. It is expected that for query
```
SELECT x, y, z
FROM R;
```
`merge_contiguous()` merges all attributes into one `ContiguousAttrs` copy group, with `bytes_to_copy` set to 12 (size of three integers).

It is also expected that for query
```
SELECT x, y, y
FROM R;
```
`merge_contiguous()` merges all attributes into two `ContiguousAttrs` copy groups, one for `x, y` and one for the last `y`.

### CODE REVIEW

File: storage/tests/SplitRowStoreTupleStorageSubBlock_unittest.cpp [hunk 1]
```diff
@@ -761,61 +761,89 @@ TEST_P(SplitRowStoreTupleStorageSubBlockTest, GetCopyGroupsForAttributeMapTest)
...
+              varlen_attrs[0].bytes_to_advance);
   } else {
     EXPECT_EQ(3, contiguous_attrs.size());
-    EXPECT_EQ(8, contiguous_attrs[0].bytes_to_copy_);
-    EXPECT_EQ(8 + size_of_string, contiguous_attrs[1].bytes_to_advance_);
+    EXPECT_EQ(8, contiguous_attrs[0].bytes_to_copy);
+    EXPECT_EQ(8 + size_of_string, contiguous_attrs[1].bytes_to_advance);
   }
+
+  // Extra test 1 for merging.
```
Comment (zuyu) [inline storage/tests/SplitRowStoreTupleStorageSubBlock_unittest.cpp]: Could we add the following comment here?

```
  // Extra test 1 for merging to simulate the following query:
  //   SELECT x, y, z FROM R;
```

Ditto for `// // Extra test 2 for merging.` below.

**Comment by zuyu** [comment]
@jianqiao Still have gcc build errors.

**Comment by zuyu** [comment]
Is `ane run of` in Line 49 of `SplitRowStoreTupleStorageSubBlock.hpp` a typo of `one run of`?

Also, we need to remove the last `_` in `bytes_to_advance_` in Line 72 and `bytes_to_copy_` in Line 101 of the same file.

**Comment by jianqiao** [comment]
All updated.

**Pull request merged**